frontend/src/components/chart_panel.jsx

```diff
@@ -91,6 +92,66 @@ const ChartPanel = ({ settings, id, setPanels }) => {
             },
         });
 
+        settings.tooltip && newChart.subscribeCrosshairMove((param) => {
```

Comment:
Doesn't it need a sort of a timeout?

Reply:
I don't think a timeout is needed here. It doesn't affect anything.